services/singleuser/Dockerfile

```diff
@@ -33,21 +30,16 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash && sudo apt-get install git-lfs=2.4.2
 USER jovyan
 
-# install pinned jupyterhub and ensure notebook is installed
+# install renku-python, jupyterlab and git extension
 RUN python3 -m pip install -U pip && \
-    python3 /tmp/install && \
-    python3 -m pip install notebook jupyter_contrib_nbextensions widgetsnbextension && \
-    jupyter contrib nbextension install --user
-
-ADD setup.py gitlab_commit_push.py /tmp/gitlab_commit_push/
-RUN python3 -m pip install /tmp/gitlab_commit_push/ && \
-    jupyter serverextension enable --py gitlab_commit_push
-
-ADD notebook.json /home/jovyan/.jupyter/nbconfig/
-ADD gitlab-commit-push.js /home/jovyan/.local/share/jupyter/nbextensions/
-
-# install renku-python
-RUN python3 -m pip install renku
+    python3 -m pip install renku && \
+    pip install -U jupyterlab && \
+    jupyter labextension update @jupyterlab/hub-extension --no-build && \
+    jupyter labextension install @jupyterlab/git --no-build && \
+    jupyter lab build && \
+    jupyter labextension list && \
+    python3 -m pip install -e git+https://github.com/SwissDataScienceCenter/jupyterlab-git.git@fix-git-current-dir#egg=jupyterlab_git && \
+    jupyter serverextension enable --py jupyterlab_git
 
 COPY git-config.bashrc /home/jovyan/
```

Comment:
where does this come from?

Reply:
?
since i update two lab extensions, i defer the build (yarn & webpack are slow...)
`jupyter labextension list` is there just to check that the extensions are ok on the build logs

Reply:
ah nevermind, I thought you added the `git-config.bashrc` file but I see it's in the directory already.